content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -60,6 +60,7 @@ their authors, not the Kubernetes team.
 | Python               | [github.com/mnubo/kubernetes-py](https://github.com/mnubo/kubernetes-py) |
 | Ruby                 | [github.com/Ch00k/kuber](https://github.com/Ch00k/kuber) |
 | Ruby                 | [github.com/abonas/kubeclient](https://github.com/abonas/kubeclient) |
+| Ruby                 | [github.com/kontena/k8s-client](github.com/kontena/k8s-client) |
```

Comment:
The link is missing the `https://` prefix.

Reply:
Good catch, fixed.